src/main/java/servlets/FetchOptionsServlet.java

```diff
@@ -61,7 +61,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       string in the given array is already capitalized, since this is the case
```

Comment:
Alternatively, you could force the capitalization of the first character.

Reply:
Was originally not going to do this because its unnecessary given the singular use case (as described in the comment:

      Assumes that the first character of every
      string in the given array is already capitalized, since this is the case
      for search strings, in which all Strings are fully upper-cased, and
      this servlet is being used to fetch those search strings.

However, given that I have changed this to static/it may be used elsewhere, I've decided to force the capitalization to make it more broadly applicable (no prior assumptions).